### PR TITLE
update-aws-ipi-rollback

### PR DESCRIPTION
**Pull request by weliang1**
https://github.com/openshift/release/pull/32093 is messed up and just close it.
Create this new one to do the same thing.

@zhaozhanqi @lihongan PTAL!

### CODE REVIEW

File: ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/sdn2ovn-rollback/cucushift-installer-rehearse-aws-ipi-ovn-sdn2ovn-rollback-workflow.metadata.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+	"path": "cucushift/installer/rehearse/aws/ipi/ovn/sdn2ovn-rollback/cucushift-installer-rehearse-aws-ipi-ovn-sdn2ovn-rollback-workflow.yaml",
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/sdn2ovn-rollback/cucushift-installer-rehearse-aws-ipi-ovn-sdn2ovn-rollback-workflow.metadata.json:2]: The same comments as https://github.com/openshift/release/pull/32093#discussion_r965441288, for easy understanding of what is the target networktype, I suggest to move the workflow to `aws/ipi/sdn` folder.

Comment (weliang1) [reply ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/sdn2ovn-rollback/cucushift-installer-rehearse-aws-ipi-ovn-sdn2ovn-rollback-workflow.metadata.json:2]: > The same comments as [#32093 (comment)](https://github.com/openshift/release/pull/32093#discussion_r965441288), for easy understanding of what is the target networktype, I suggest to move the workflow to `aws/ipi/sdn` folder.

@zhaozhanqi How about to keep sdn2ovn migration in aws/ipi/ovn and rollback in aws/ipi/sdn? WDYT?

Comment (weliang1) [reply ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/sdn2ovn-rollback/cucushift-installer-rehearse-aws-ipi-ovn-sdn2ovn-rollback-workflow.metadata.json]: @jianlinliu Move the workflow to aws/ipi/sdn folder now.

**Pull request merged**